JS/index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>jquery practice</title>
+    <script type="text/javascript" src="https://code.jquery.com/jquery-3.6.0.min.js"></script>
+</head>
+
+<body>
+    <h1>The Burger Joint</h1>
+
+    <ul>
+        <li id="burger">Frito Pie Burger</li>
```

Comment:
Should this be class not I'd? Burger seems general.

Reply:
For this particular file, I was just going off of what John was doing in class. I replicated the exercises he had told us to  do including the classes and ids. It's merely a 'follow-along' file. 
The indentations are not part of the follow-along; I will fix those on a new branch.